### ISSUE TITLE
Footprint 12BH611-GR: wrong pins

### ISSUE THREAD

**Issue by frarees** [KiCad/kicad-footprints#1113]
I used 12BH611-GR as a footprint for a Device/Battery. 

12BH611-GR, from a top view, having the snap on the right side:

- Pin 1 is used for the negative side (upper)
- Pin 2 is used for the positive side (lower)

But in order to work with Device/Battery, it should be:

- Pin 1: positive side (lower)
- Pin 2: negative side (upper)

I went ahead and changed it in my local copy, and verified that everything worked the way it should after swapping the pin numbers.

**Comment by evanshultz** [comment]
Thanks for reporting this potential issue. I see this is your first post here, and I'm sorry it's because of a problem but thanks for taking the time to let us know!

The datasheet is always helpful so we can all understand the problem. I found https://www.mouser.com/datasheet/2/209/EPD-200766-1274481.pdf and it shows this:
![image](https://user-images.githubusercontent.com/1936989/49027483-2c93b200-f198-11e8-8f57-ad3a57a6168a.png)

Here is the footprint in our current library:
![image](https://user-images.githubusercontent.com/1936989/49027453-1d146900-f198-11e8-99f6-35eb4718c8da.png)

Both have the snap on the left side and the pinout matches with negative on the top.

Also note that pin 1 is negative.

And here the schematic symbol that would go with this footprint you mentioned above (Device:Battery_Cell is the same pinout):
![image](https://user-images.githubusercontent.com/1936989/49027588-6a90d600-f198-11e8-916a-e8947d11e964.png)

Pin 1 is positive. So that's the issue. The pin numbers of the pins on this footprint need to be swapped.

I checked all other footprints with a "+" marked in them (I didn't check unmarked ones) and pin 1 is positive elsewhere, so this is the only broken footprint I found.

Sorry that you ran into this issue but thanks again for reporting it. Are you willing to make a PR to fix this?
